src/components/editor-load-error/index.jsx

```diff
@@ -12,51 +11,46 @@ import './style.scss';
 /**
  * Displays an error notice when the editor fails to load.
  *
+ * Simple error message avoids `@wordpress` components to avoid
+ * complex dependency management during editor initialization.
```

Comment:
This simple error message is used within the editor entry file if an error occurs during the sequence to load and initialize foundational `@wordpress` modules. Libraries like `react` or `@wordpress/components` may not be available depending on the error, so we use string templates instead. 

Long term, we should broadcast the editor load failure to the native host app and allow it to control how an error is presented to the user.

Reply:
> we should broadcast the editor load failure to the native host app and allow it to control how an error is presented to the user

That sounds great. And thanks for addressing it. It already helped me see an error due to the new dependencies.